istioctl/cmd/auth.go

```diff
@@ -90,7 +90,7 @@ THIS COMMAND IS STILL UNDER ACTIVE DEVELOPMENT AND NOT READY FOR PRODUCTION USE.
 	}
 
 	upgradeCmd = &cobra.Command{
-		Use:   "upgrade -f <yaml-file>",
```

Comment:
Don't we need the `-f` anymore?

Reply:
I think `Use` ignore hardcoded flags? `TestAuthUpgrade` from `auth_test.go` still passes. Plus, many places do not put flags in `Use`.
https://github.com/istio/istio/blob/master/istioctl/cmd/convert_ingress.go#L185

Reply:
I mean what's the point of removing `-f` but leaving `<yaml-file>` there?

Reply:
Also it's better to update the `Example` to call out it supports multiple files.

Reply:
Updated for both. PTAL.